<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Venta</class>
 <widget class="QDialog" name="Venta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1" colspan="4">
      <widget class="QLabel" name="label_dia">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QPushButton" name="Boton_10">
       <property name="text">
        <string>10%</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Socio cinemamax</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QPushButton" name="Boton_25">
       <property name="text">
        <string>25%</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Horario</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" rowspan="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Descuentos</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QListWidget" name="listWidget_horarios"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Cantidad</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QRadioButton" name="radioButton_2d">
       <property name="text">
        <string>2D</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Con tarjeta de credito</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Dia</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Con tarjeta de debito</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QComboBox" name="comboBox_pelicula"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Seleccione la pelicula</string>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="QPushButton" name="Boton_2x1">
       <property name="text">
        <string>2x1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Tipo de entrada</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QRadioButton" name="radioButton_3d">
       <property name="text">
        <string>3D</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="spinBox_cantidad"/>
     </item>
     <item row="12" column="3">
      <widget class="QPushButton" name="Boton_continuar">
       <property name="text">
        <string>Continuar</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Total</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="label_total">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
